test/cucumber5.spec.js

```diff
@@ -4,7 +4,7 @@ describe('cucumber version 5', function() {
   it('runs successful features', function() {
     return util
       .runOne(
-        'test/cucumber/conf/cucumber4Conf.js --cucumberOpts.tags @cucumber4'
+        'test/cucumber/conf/cucumber4-5-6Conf.js --cucumberOpts.tags @cucumber4'
```

Comment:
Is the tag here supposed to be cucumber5 ?

Reply:
No. It is only used in `test/cucumber/features/cucumber4.feature`.
It could maybe be renamed `@cucumber4-5-6`

Reply:
I would suggest adding 2 new feature files https://github.com/protractor-cucumber-framework/protractor-cucumber-framework/tree/master/test/cucumber/features

Reply:
With the exact same content ?
This would generate a lot of code duplication :(
What are your reasons for doing so ?

Reply:
To follow convetion I would say yes cucumber 6 is a new feature so it should have it's own feature file. The code duplication is very small and it is better for regression testing going forward. I would say it's bad practice but If you feel strongly about it I would say the least that could be done is add a cucumber 5 and cucumber 6 tag to the cucumber 4 feature

Reply:
Done

Reply:
Awesome thanks, I approved